Author: Unrud <unrud@outlook.com>
Description: restarting fails, if Python 2 is not installed at /usr/bin/python
Last-Update: 2019-05-11
--- a/src/FretsOnFire.py
+++ b/src/FretsOnFire.py
@@ -98,10 +98,7 @@
           else:
             os.execl("./FretsOnFire", "./FretsOnFire", *sys.argv[1:])
         else:
-          if os.name == "nt":
-            bin = "c:/python25/python"
-          else:
-            bin = "/usr/bin/python"
+          bin = sys.executable
           os.execl(bin, bin, "FretsOnFire.py", *sys.argv[1:])
       except:
         Log.warn("Restart failed.")
